### ISSUE TITLE
Web - Bookmark List - Missing ALT Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#77]
The SVG left-pointing arrow is missing a title, description, or some piece of ARIA as part of the Home Page link. There is no ALT text or some kind of description for the SVG images of the bookmark icon which should tell the user which bookmark they are deleting. There are a bunch of other images most likely in the footer that I couldn’t see without having my tools in IE to find that are empty or missing ALT text. 

There might be missing ALT text on other pages that I didn't have time to check so please those too.

Bookmark List - https://dolcontentdev.opadev.dol.gov/ilab/complychain/build/bookmarks

**Comment by KyleLivingstonDOL** [comment]
ZTI, can we look into the SVg images in the bookmark icons?

**Comment by Abychalil** [comment]
alt text provided for mentioned items including the four items on the footer menu

**Comment by jfc3-dol** [comment]
@Abychalil,

Thanks for adding the necessary ALT text. Will look at it when the next version/build is sent, and I have time to test.

**Comment by jfc3-dol** [comment]
There is now ALT text on the left pointing arrow, so I'm going to close the issue.
